metadataservice/schema/run_start.json

```diff
@@ -9,7 +9,7 @@
             "description": "Name of project that this run is part of"
         },
         "sample": {
-            "type": ["object"],
+            "type": ["string"],
```

Comment:
In the [corresponding line the in bluesky PR](https://github.com/NSLS-II/bluesky/pull/375/files#diff-7f868e2356e2827e8024a1cba664c267R8), I allowed it to be a string or an objec*. Changing it to string would require a migration, and I don't want to do that anyway. (P.S. I know you squeezed this work in amidst school stuff -- no worries.)